apps/cow-fi/components/ChartSection/index.tsx

```diff
@@ -25,7 +25,7 @@ export function ChartSection({ platforms }: ChartSectionProps) {
     return null
   }, [platforms.ethereum.contractAddress])
 
-  const { data, loading, error } = useQuery(tokenPriceQuery, {
+  const { data, loading } = useQuery(tokenPriceQuery, {
```

Comment:
I guess it wold be ideal to handle the error rather than remove it, although I guess this PR was to cleanup only...